examples/kafka/serverlesscluster.yaml

```
@@ -0,0 +1,85 @@
+apiVersion: kafka.aws.upbound.io/v1beta1
+kind: ServerlessCluster
+metadata:
+  annotations:
+    meta.upbound.io/example-id: kafka/v1beta1/serverlesscluster
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    clientAuthentication:
+    - sasl:
+      - iam:
+        - enabled: true
+    clusterName: Example
+    region: us-east-2
+    vpcConfig:
+      - securityGroupIdSelector:
+          matchLabels:
+            testing.upbound.io/example-name: sg
+        subnetIdSelector:
+          matchLabels:
+            testing.upbound.io/example-name: subnets
+---
+
+apiVersion: ec2.aws.upbound.io/v1beta1
+kind: SecurityGroup
+metadata:
```

```suggestion
metadata:
  annotations:
    meta.upbound.io/example-id: kafka/v1beta1/serverlesscluster
```
Please add the example-id.